src/videojs-hls.js

```diff
@@ -10,6 +10,9 @@ var
   // a fudge factor to apply to advertised playlist bitrates to account for
   // temporary flucations in client bandwidth
   bandwidthVariance = 1.1,
+
+  // the amount of time to wait between checking the state of the buffer
+  bufferCheckInterval = 500,
```

Comment:
why 500?
This PR unhooks listening to timeupdate to run fillbuffer right? That used to run at either 250 or less.